util.go

```diff
@@ -20,6 +20,9 @@ const (
 // nextPowOf2 computes the next highest power of 2 of 32-bit v.
 // Source: https://graphics.stanford.edu/~seander/bithacks.html#RoundUpPowerOf2
 func nextPowOf2(v uint32) uint32 {
+	if v == 0 {
+		return 2
```

Comment:
Is there a reason not to use 1 as the return value for 0 input instead of 2? After all `nextPowOf2(1) == 1`.

Reply:
😁 done, `nextPowOf2(1) == 1`